### PR TITLE
Feature/add programs tool

### PR DESCRIPTION
**Pull request by idelcano**
A script to make easy update the departaments list.
check readme

### CODE REVIEW

File: tools/departament_list_editor.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
+from datetime import datetime
+import requests
+import csv
+import base64
+
+from requests.auth import HTTPBasicAuth
+
+import argparse
+import json
+import csv
```
Comment (nshandra) [inline tools/departament_list_editor.py]: Duplicated and unused imports

File: tools/departament_list_editor.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        if args.convert.endswith('.csv'):
+            convertCSV2JSON(args.convert, output_filename+".json")
+        else:
+            convertJSON2CSV(args.convert, output_filename+".csv")
+    else:
+        validate_download_args(args)   
+        creedentials = f"{args.user}:{args.password}"
+        creedentials = base64.b64encode(creedentials.encode()).decode()
+        headers = {
+            'Authorization': f'Basic {creedentials}'
```
Comment (nshandra) [inline tools/departament_list_editor.py]: typo in creedentials

File: tools/departament_list_editor.py (hunk 1 above)
Comment (nshandra) [reply tools/departament_list_editor.py]: Done

File: tools/departament_list_editor.py (hunk 2 above)
Comment (nshandra) [reply tools/departament_list_editor.py]: Done

**Review by nshandra — APPROVED** [review]
Fixed identified issues, added formatting and file exception handling. Added section headers to README.

Ready to merge.

**Review by ifoche — APPROVED** [review]
thanks @idelcano @nshandra

**Pull request merged**